src/tests/ftest/daos_test/daos_core_test-rebuild.py

```diff
@@ -32,6 +32,12 @@ class DaosCoreTestRebuild(DaosCoreBase):
     :avocado: recursive
     """
 
+    # Test variants that should be skipped
+    CANCEL_FOR_TICKET = [
+        ["DAOS-5851", "test_name", "rebuild tests 0-10"]
+        ["DAOS-6442", "test_name", "rebuild tests 18"]
```

Comment:
(pylint-invalid-sequence-index) Sequence index is not an int, slice, or instance with __index__

Reply:
Sorry I forgot the comma in my suggestion.  It should be:
```suggestion
        ["DAOS-5851", "test_name", "rebuild tests 0-10"],
        ["DAOS-6442", "test_name", "rebuild tests 18"]
```